lib/auth/apiserver.go

```diff
@@ -622,20 +625,23 @@ func (s *APIServer) generateToken(auth ClientI, w http.ResponseWriter, r *http.R
 }
 
 type registerUsingTokenReq struct {
-	HostID string        `json:"hostID"`
-	Role   teleport.Role `json:"role"`
-	Token  string        `json:"token"`
+	HostID   string        `json:"hostID"`
```

Comment:
we are using `_` vs camel case in Teleport, `hostID` is unfortunate, so we have to keep it, but the rest of the args let's keep them underscore for now